api/v1alpha1/jobset_types.go

```diff
@@ -19,11 +19,11 @@ import (
 )
 
 const (
-	JobSetKey        string = "jobset.sigs.k8s.io/jobset"
+	JobSetKey        string = "jobset.sigs.k8s.io/jobset-name"
 	JobIndexKey      string = "jobset.sigs.k8s.io/job-index"
-	ReplicasKey      string = "jobset.sigs.k8s.io/job-replicas"
+	ReplicasKey      string = "jobset.sigs.k8s.io/replicatedjob-replicas"
 	RestartsKey      string = "jobset.sigs.k8s.io/restart-attempt"
```

Comment:
just noticed this, I don't think we should put it in the api, this is an implementation detail.

Reply:
Ok I moved the RestartsKey constant to the controller.